packages/datagrid/src/components/DefaultHeaderRenderer/QualityBar.component.js

```diff
@@ -2,11 +2,12 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 import { useTranslation } from 'react-i18next';
-
+import { Button } from 'react-bootstrap';
```

Comment:
Curious there, why do we use Button from react-bootstrap instead of Action in react-components ?